README.md

```diff
@@ -63,29 +68,80 @@ sim_log_data(path_to_synthetic_log_data)
+# end date of trajectories
+end_date = datetime.date(2021, 10, 5)
+# api key for openroute service, generated from https://openrouteservice.org/
+api_key = "mock_api_key"
+# Length of off-cycle + length of on-cycle in minutes
+cycle = 15
+# Length off-cycle / (length off-cycle + length on-cycle)
+percentage = 0.8
+# dictionary of personal attributes for each user, set to None if random, check Attributes class for usage in simulate_gps_data module.
+personal_attributes = {
+    "User 1":
+    {
+        "main_employment": "none", 
+        "vehicle" : "car",
+        "travelling_status": 10,
+        "active_status": 0
+    },
+
+    "Users 2-4":
+    {
+        "main_employment": "university",
+        "vehicle" : "bicycle",
+        "travelling_status": 8,
+        "active_status": 8,
+        "active_status-16": 2 
+    },
+
+    "User 5":
+    {
+        "main_employment": "office",
+        "vehicle" : "foot",
+        "travelling_status": 9,
+        "travelling_status-20": 1,
+        "preferred_exits": ["cafe", "bar", "cinema"] 
+    }
+}
+sample_gps_data = sim_gps_data(n_persons, location, start_date, end_date, cycle, percentage, api_key, personal_attributes)
+# save data in format of csv files
+gps_to_csv(sample_gps_data, path_to_synthetic_gps_data, start_date, end_date)
 
 # 2. Specify parameters for imputation 
 # See https://github.com/onnela-lab/forest/wiki/Jasmine-documentation#input for details
-tz_str = "America/New_York" # time zone where the study took place (assumes that all participants were always in this time zone)
-option = "daily" # Generate summary metrics "hourly", "daily" or "both"
-save_traj = False # Save imputed trajectories?
-time_start = None 
-time_end = None
-beiwe_id = None
+# time zone where the study took place (assumes that all participants were always in this time zone)
+tz_str = "America/New_York"
+# Generate summary metrics Frequency.HOURLY, Frequency.DAILY or Frequency.BOTH
+frequency = Frequency.DAILY
+# Save imputed trajectories?
+save_traj = False
+# Hyperparameters class for imputation (default leave None), from forest.jasmine.traj2stats import Hyperparameters
 parameters = None
-all_memory_dict = None
-all_BV_set= None
+# list of locations to track if visited, leave None if don't want these summary statistics
+places_of_interest = ['cafe', 'bar', 'hospital']
+# True if want to save a log of all locations and attributes of those locations visited
+save_log = True
+# threshold of time spent in a location to count as being in that location, in minutes
+threshold = 15
 
 # 3. Impute location data and generate mobility summary metrics using the simulated data above
-gps_stats_main(path_to_synthetic_gps_data, path_to_gps_summary, tz_str, option, save_traj, time_start, time_end, beiwe_id, parameters, all_memory_dict, all_BV_set)
+gps_stats_main(path_to_synthetic_gps_data, path_to_gps_summary, tz_str, frequency, save_traj, parameters, places_of_interest, save_log, threshold)
```

Comment:
I get the following output and no summary when I run this:
```
>>> gps_stats_main(path_to_synthetic_gps_data, path_to_gps_summary, tz_str, frequency, save_traj, parameters, places_of_interest, save_log, threshold)
User: user_2
GPS data are not collected or the data quality is too low
User: user_1
Read in the csv files ...
Collapse data within 10 second intervals ...
Extract flights and pauses ...
Infer unclassified windows ...
Merge consecutive pauses and bridge gaps ...
Selecting basis vectors ...
Imputing missing trajectories ...
An error occured when processing the data
```

Reply:
I'm not getting any errors when running the code. It could be a runtime error from the API. Do you think we should remove that try except statement in the summary statistics that basically just returns that message for any error, so we can have more comprehensive error messages?

Reply:
Yeah, it would be great to have more informative error messages. That `try` block in `gps_stats_main()` is unnecessarily large and broad too.